src/app/analyze/stack/stack-details/stack-details.component.ts

```diff
@@ -72,9 +58,28 @@ export class StackDetailsComponent implements OnInit {
     private logger: Logger
   ) { }
 
+  private setStackAnalysisRawData(): void {
```

Comment:
You should consider moving this out to a model. Though a private model doesn't make a lot of sense so if it must be private then maybe keep it here.